### ISSUE TITLE
[Next] Is broken on Safari/IOS

### ISSUE THREAD

**Issue by fpliger** [pyscript/pyscript#1648]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

Current `main` (the foundation of the release  `2023.09.1`) doesn't work on IOS/Safari (and any browser that doesn't support Atomics). This will probably self solve as browsers catch up but in the meantime we need a polyfill or detection of some sort...

To reproduce open this in Safari: https://fpliger.pyscriptapps.com/me35-1-micropython-copy/latest/

### Console info

```shell
Error 500: Failed to load resource: the server responded with a status of 500 (Internal Server Error)
```

**Comment by WebReflection** [comment]
> Current `main`

did you mean `next` ?

> doesn't work on IOS/Safari (and any browser that doesn't support Atomics)

Safari supports SharedArrayBuffer and all tests work to me in Safari (as long as SharedArrayBuffer is not explicitly disabled)

> `Error 500: Failed to load resource: the server responded with a status of 500 (Internal Server Error)`

This is not our project fault, rather an issue with https://esm.sh/@pyscript/core@0.1.5/core.js ... maybe I should just switch all demo to use directly either `unpkg.com` or `jsdlvr` CDN ??? The back-end does some smart assumption and breaks intentionally the code throwing an error out of the box for no reasons:

```js
/* esm.sh - error */
throw new Error("[esm.sh] " + "esbuild: Top-level await is not available in the configured target environment (\"es2021\")");
export default null;
```

I will try to file an issue though as `esm.sh` shouldn't deliberately make assumptions or break otherwise working scripts

**Comment by WebReflection** [comment]
@fpliger can you please try `https://unpkg.com/@pyscript/core` instead as module?

**Comment by fpliger** [comment]
It seems to raise a different error on Safari: `ReferenceError: Can't find variable: Worker`, no matter if we specify `<py-script src="...">` or `<py-script worker="...">`

check it [out here](https://fpliger.pyscriptapps.com/no-blocking-no-stress-copy/latest/)

**Comment by WebReflection** [comment]
your link works just fine in here

![Screenshot from 2023-08-28 18-51-35](https://github.com/pyscript/pyscript/assets/85749/edd67690-e196-42ca-ad83-1c3f89137461)

**Comment by WebReflection** [comment]
@fpliger if you're testing on Epiphany or GNOME Web please don't ... use the provided Safari (Webkit TP) official build outof flatpak https://webkit.org/downloads/ (see bottom page)

**Comment by WebReflection** [comment]
btw, they suggested to use https://esm.sh/@pyscript/core@0.1.5/core.js?target=esnext instead to avoid the issue ... that still stinks to me as resolution, explained in the filed bug.

**Comment by fpliger** [comment]
@WebReflection , I'm using Safari on Mac ... and still no luck :( (still seeing `ReferenceError: Can't find variable: Worker` ) Weird. 

We can check it together tomorrow...

**Comment by WebReflection** [comment]
> I'm using Safari on Mac

which version?

**Comment by fpliger** [comment]
I'm on `Version 16.3 (18614.4.6.1.6)`. Have tested with other 3 people and they also have issues (not sure about their version though)

**Comment by fpliger** [comment]
This has been fixed by #1655 and changes upstream. Closing.

@WebReflection feel free to re-open if I'm missing something
